packages/e2e-test-utils-playwright/src/request-utils/comments.ts

```diff
@@ -7,20 +7,35 @@ export interface Comment {
 	id: number;
 	author: number;
 	content: string;
+	post: number;
 }
 
 /**
  * Create new comment using the REST API.
  *
- * @param {} this    RequestUtils.
- * @param {} comment Comment.
+ * It uses the current user if no author is defined inside `comment`.
+ *
+ * @param {} this           RequestUtils.
+ * @param {} comment        Partial<Comment>.
  */
-export async function createComment( this: RequestUtils, comment: Comment ) {
-	this.rest( {
+export async function createComment(
+	this: RequestUtils,
+	comment: Partial< Comment >
```

Comment:
Nit: I'm not sure if `Partial` is the right type here to represent the payload. Isn't there some field actually required to create a comment?